ibm_platform_services/iam_identity_v1.py

```diff
@@ -40,7 +40,7 @@
 class IamIdentityV1(BaseService):
     """The iam_identity V1 service."""
 
-    DEFAULT_SERVICE_URL = 'https://iam.cloud.ibm.com'
+    DEFAULT_SERVICE_URL = 'https://iam.test.cloud.ibm.com'
```

Comment:
The default service url always should point to the production environment.

Reply:
This most likely means that the "test" version of the API definition was used to generated the SDK code.  Need to be pretty careful about that :)

Reply:
yes thats correct, but our staging and prod both endpoints work the same way, all the response model are same for both envs. however at one place I have missed to remove "test" :) (now corrected)